serverless/tensorflow/mediapipe/nuclio/function.yaml

```diff
@@ -0,0 +1,90 @@
+# CONFIG
+metadata:
+  name: mediapipe-pose
+  namespace: cvat
+  annotations:
+    name: Mediapipe human pose estimation
+    type: detector
+    framework: tensorflow
+    spec: |
+      [
+        {
+          "name" : "body",
+          "type" : "skeleton",
```

Comment:
Could you please add "svg" as well? Please look here: https://github.com/opencv/cvat/blob/develop/serverless/pytorch/mmpose/hrnet32/nuclio/function.yaml. You can contract it in CVAT UI and download: https://opencv.github.io/cvat/docs/manual/advanced/annotation-with-skeletons/creating-the-skeleton/

Now we can create a label from a model. UI has a new easy way to create such complex labels because your model can provide it out of the box.